bin/oscapd-cli

```diff
@@ -30,6 +30,78 @@ import sys
 from datetime import datetime
 import os.path
 
+class TaskAccessor:
+
+    def __init__(self):
+        self._attributes = dict()
+
+    @staticmethod
+    def get_bool(values):
+        TaskAccessor._expect_param_len(values, 1)
+        bool_val = TaskAccessor._parse_bool(values[0])
+        return [bool_val,]
+
+    @staticmethod
+    def _parse_bool(val):
```

Comment:
As we discussed, please refactor this so that we can use a single bool parsing function everywhere.